### ISSUE TITLE
Mac OS build of galaxy fails on pysam 

### ISSUE THREAD

**Issue by cganote** [galaxyproject/galaxy#6423]
Pip fails to install pysam on MacOS. Possibly related to:
https://github.com/pysam-developers/pysam/issues/682
Here's what I see:
https://gist.github.com/cganote/2c557a3bd060773078d45fb311164918

My software installs:
MacOSX 10.9.5
Python is Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 12:54:16)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
pip 10.0.1
virtualenv 16.0.0
Samtools Version: 1.2 (using htslib 1.2.1)
-Carrie

**Comment by machristie** [comment]
This failed for me too.  I tried to workaround it by [downloading the source for pysam-0.15.1](https://github.com/pysam-developers/pysam/releases/tag/v0.15.1) and running `python setup.py install` with the Galaxy .venv activated:

```
mv ~/Downloads/pysam-0.15.1.tar.gz .
tar zxf pysam-0.15.1.tar.gz 
cd pysam-0.15.1/
pip install -r requirements.txt 
python setup.py install
```


This built pysam but unfortunately that didn't quite work for galaxy because the version wasn't updated until after this tagged commit. So `sh run.sh` is still trying and failing to install pysam-0.15.1.

So I downloaded the [sources at commit 86694a40](https://github.com/pysam-developers/pysam/archive/86694a40bb73770d3cd207e8f02365bf99052bf9.zip) and tried again.

```
mv ~/Downloads/pysam-86694a40bb73770d3cd207e8f02365bf99052bf9.zip .
unzip pysam-86694a40bb73770d3cd207e8f02365bf99052bf9.zip 
cd pysam-86694a40bb73770d3cd207e8f02365bf99052bf9/
pip install -r requirements.txt 
python setup.py install
```

And this successfully installed pysam-0.15.1 into the Galaxy .venv virtual environment.
